### ISSUE TITLE
SignInAsync crashes on UWP SUA if Xbox One S not connected to the internet

### ISSUE THREAD

**Issue by FREEZX** [Microsoft/xbox-live-api#508]
SignInAsync crashes the application in a UWP SUA environment on an Xbox one S with the latest release (1806 QFE 3), and when building from the latest master at https://github.com/Microsoft/xbox-live-api/commit/2166ffcb4a1ca1f5d89a43dce9ab4c5ff9b797b7 

To make sure, i made a small repro project with my build from master by replacing the .dll and .winmd files of the unity plugin. Everything works just fine if there is connection, but if the xbox is disconnected, the app crashes after a few seconds after calling SignInAsync.

Here's the project if you want to try it out for yourself:
https://drive.google.com/open?id=1BqfLjVkXVGnWO_m0RRQGAZz7qnvMnn4h
The built unity project can be found in Builds/WSA

Regards

**Comment by jasonsandlin** [comment]
Thanks for the bug report.  We'll investigate soon

**Comment by jasonsandlin** [comment]
We tried our UWP samples that don't use Unity and couldn't reproduce the issue.  We believe this might be an issue with the Xbox Live Unity plugin.  Are you using https://github.com/Microsoft/xbox-live-unity-plugin?

**Comment by FREEZX** [comment]
No, i'm using this project with the .NET scripting backend of Unity (at the time of reporting, i was using some version of Unity 2017.4.x).

**Comment by jasonsandlin** [comment]
Since your building against the source already can you switch to branch 1804qfe and try again?  With this source, you won't need to include libHttpClient if that's in your project.

**Comment by jasonsandlin** [comment]
Ok, good - you can ship with this.  We'll be updating our latest release soon to this branch to avoid  instability found in our recent release

**Comment by Murtaya** [comment]
ATG (https://forums.xboxlive.com/questions/84717/index.html) directed me here since the 1806 instability issue was causing my game to fail cert (crashing on sign in for new users).

Has the latest release branch (1807) been updated to fix the instability issues?  I noticed it was updated a day after Jason's last comment, but I'm still noticing a sign-in crash when the console is disconnected from the internet (I assume the same as what FREEZX was seeing?).  

Both 1804 and 1711 seem to not have the crashing issue (I was a bit confused though since the 1804 and 1807 packages look the same--even Unity didn't think there was a difference, but 1804 does behave differently), so I'm gonna try going through cert with 1804.  

Thanks!
Mike
